src/Library/demos/Button/code.rs

```diff
@@ -0,0 +1,29 @@
+use crate::workbench;
+use adw::prelude::*;
+
+pub fn main() {
+    let button_ids: Vec<&str> = vec![
+        "regular",
+        "flat",
+        "suggested",
+        "destructive",
+        "custom",
+        "disabled",
+        "circular-plus",
+        "circular-minus",
+        "pill",
+        "osd-left",
+        "osd-right",
+    ];
+
+    for id in button_ids {
+        let button: gtk::Button = workbench::builder()
+            .object(id)
+            .expect("Button need to be present.");
```

Comment:
```suggestion
            .unwrap();
```
I moved to unwrap for the newer examples